.github/workflows/elixir.yml

```diff
@@ -104,3 +104,39 @@ jobs:
         run: mix format --check-formatted
       - name: Test
         run: mix test
+  opentelemetry-oban:
+    needs: [test-matrix]
+    if: (contains(github.event.pull_request.labels.*.name, 'elixir') && contains(github.event.pull_request.labels.*.name, 'opentelemetry_oban'))
+    env:
+      app: 'opentelemetry_oban'
+    defaults:
+      run:
+        working-directory: instrumentation/${{ env.app }}
+    runs-on: ubuntu-18.04
+    name: Opentelemetry Oban test on Elixir ${{ matrix.elixir_version }} (OTP ${{ matrix.otp_version }})
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.test-matrix.outputs.matrix) }}
```

Comment:
I don't see any setup of postgres for your tests. You can copy from the ecto tests.

https://github.com/open-telemetry/opentelemetry-erlang-contrib/pull/6/files#diff-567a3fdf49d6f0f0119dda6d1c098fcb5ee3785ee956481b3daaf5a2f246076fR37-R45